pkg/util/ipvs/kernelcheck_linux.go

```diff
@@ -63,29 +64,46 @@ func (r RequiredIPVSKernelModulesAvailableCheck) Check() (warnings, errors []err
 	loadModules.Insert(mods...)
 	modules := wantModules.Difference(loadModules).UnsortedList()
 
-	// Check builtin modules exist or not
-	if len(modules) != 0 {
-		builtinModsFilePath := fmt.Sprintf("/lib/modules/%s/modules.builtin", kernelVersion)
-		out, err := r.Executor.Command("cut", "-f1", "-d", " ", builtinModsFilePath).CombinedOutput()
+	if len(modules) == 0 {
+		return nil, nil
+	}
+
+	// Check if loadable modules are installed
+	var missingModules []string
+	for _, module := range modules {
+		out, err = r.Executor.Command("modinfo", module).CombinedOutput()
 		if err != nil {
-			errors = append(errors, fmt.Errorf("error getting required builtin kernel modules: %v(%s)", err, out))
-			return nil, errors
+			warnings = append(warnings, fmt.Errorf("error getting module info for %q; error: %v; output: %s", module, err, strings.TrimSpace(string(out))))
```

Comment:
we decided to use pkg-error everywhere in the meantime:
https://godoc.org/github.com/pkg/errors

-->
```
errors.Wrapf(err, "error getting module info for %q; output: %s; error:", module, strings.TrimSpace(string(out)))
```